portal_tables/sync_publications.py

```diff
@@ -34,81 +37,109 @@ def add_missing_info(pubs, grants):
-        'publicationAuthors', 'publicationAbstract', 'publicationAssay', 
-        'publicationTumorType', 'publicationTissue', 'publicationThemeName',
-        'publicationConsortiumName', 'publicationGrantNumber', 'grantName',
-        'publicationDatasetAlias', 'publicationAccessibility'
+        'Publication Doi', 'Publication Journal', 'Pubmed Id', 'Pubmed Url',
+        'Link', 'Publication Title', 'Publication Year', 'Publication Keywords',
+        'Publication Authors', 'Publication Abstract', 'Publication Assay', 
+        'Publication Tumor Type', 'Publication Tissue', 'theme',
+        'consortium', 'Publication Grant Number', 'grantName',
+        'Publication Dataset Alias', 'Publication Accessibility',
+        'entityId'
     ]
     pubs = pubs[col_order]
 
-    # Convert list column into string to match with table schema.
-    pubs.loc[:, 'publicationDatasetAlias'] = (
-        pubs['publicationDatasetAlias']
-        .str.join(", ")
-    )
+    if dryrun:
+        print(u"\u26A0", "WARNING:",
+            "dryrun is enabled (no updates will be done)\n")
 
-    new_rows = pubs.values.tolist()
-    syn.store(Table(schema, new_rows))
+    else:
+        print("Synchronizing publications staging database to production database...\n", pubs)
+        table_rows = pubs.values.tolist()
+        syn.store(Table(schema, table_rows))
 
+    return pubs
 
 def main():
     """Main function."""
     syn = utils.syn_login()
     args = get_args()
 
-    manifest = (
-        syn.tableQuery(f"SELECT * FROM {args.manifest}")
-        .asDataFrame()
-        .fillna("")
+    new_cols = [
+        'theme',
+        'consortium',
+        'grantName'
+    ]
+
+    if args.dryrun:
+        print(
+        "Inputs will be processed and provided for review", 
+        "\n\nDatabase will NOT be updated."
     )
-    curr_pubs = (
-        syn.tableQuery(f"SELECT pubMedId FROM {args.portal_table}")
-        .asDataFrame()
-        .pubMedId
-        .to_list()
+    
+    manifest = (
+        pd.read_csv(
+        syn.get(args.manifest).path,
+        header=0).fillna("")
     )
 
-    # Only add pubs not currently in the Publications table.
-    new_pubs = manifest[~manifest['pubmedId'].isin(curr_pubs)]
-    if new_pubs.empty:
-        print("No new publications found!")
-    else:
-        print(f"{len(new_pubs)} new publications found!\n")
-        if args.dryrun:
-            print(u"\u26A0", "WARNING:",
-                  "dryrun is enabled (no updates will be done)\n")
-            print(new_pubs)
-        else:
-            print("Adding new publications...")
-            grants = (
-                syn.tableQuery(
-                    "SELECT grantNumber, grantName FROM syn21918972")
-                .asDataFrame()
-            )
-            new_pubs = add_missing_info(new_pubs.copy(), grants)
-            sync_table(syn, new_pubs.copy(), args.portal_table)
-    print("DONE ✓")
+    print("\n\nCSV downloaded from Synapse:\n", manifest)
```

Comment:
Do you think it'd be valuable to add another flag (maybe something like `-v`, `--verbose`) and if passed, then output the `manifest`, `database`, and `new_table`?  Otherwise, only print:

```
$ python portal_tables/sync_publications.py
Inputs will be processed and provided for review 

Database will NOT be updated.

Processing publications staging database...

Copy of final table stored at: ./final_table.csv
DONE ✓
```